Makefile

```diff
@@ -3,6 +3,13 @@ export GO111MODULE=on
 vet:
 	@go vet -copylocks ./...
 
+fmt:
+	@res=$$(go fmt ./...); \
+	if [ -n "$${res}" ]; then \
+		echo "go fmt failed on these files:"; echo "$${res}"; echo; \
```

Comment:
Do you need to put the @ in front of the echos in order for make to not output the commands first?

Reply:
this doesn't output the echos as currently written.